src/ui/components/filters/filteroptionscomponent.js

```diff
@@ -32,7 +33,7 @@ const OptionTypes = {
 };
 
 class FilterOptionsConfig {
-  constructor (config) {
+  constructor (config, persistedFilter, persistedLocationRadius) {
```

Comment:
Nit: Only one or the other of these `persistedX` variables should be non-null at any time right? It makes me feel like this can be factored out into a `persistedNode` as one parameter. Hyf?

Reply:
They can both be non-null at the same time, though each FilterOptions will only use one or the other depending on whether it's a RADIUS_FILTER or STATIC_FILTER. I can combine them into a single object

Reply:
Okay, updated!